whotracksme/website/plotting/plots.py

```diff
@@ -68,14 +74,15 @@ def doughnut_chart(values, labels, name, color_palette=False):
         hoverinfo="label+percent",
         hole=0.65,
         pull=0.07,
-        sort=not(color_palette),
+        sort=not (color_palette),
         textinfo="label",
         textfont=dict(
             family=CliqzFonts.regular,
             color=CliqzColors["white"]
         ),
         marker=dict(
-            colors=palette(CliqzColors["blue"], CliqzColors["purple"], len(labels)) if color_palette else [CliqzColors["blue"], CliqzColors["purple"]],
+            colors=palette(CliqzColors["blue"], CliqzColors["purple"], len(labels)) if color_palette else [
```

Comment:
If `CliqzColors` is not a class, then it should not be CamelCase (PEP8 styling guidelines). (likewise `CliqzFonts` -> `cliqz_fonts`)